docs/source/archive/gsoc/reports/2025/vulnerablecode_michael.rst

```diff
@@ -0,0 +1,224 @@
+
+Linked Pull Requests
+--------------------
+
+.. list-table::
+   :widths: 10 40 20
+   :header-rows: 1
+
+   * - Sr. no
+     - Name
+     - Link
+   * - 1
+     - Add Live Evaluation API endpoint and PyPa live pipeline importer
+     - `aboutcode-org/vulnerablecode#1969
+       <https://github.com/aboutcode-org/vulnerablecode/pull/1969>`_
+   * - 2
+     - Add Gitlab Live V2 Importer
+     - `aboutcode-org/vulnerablecode#1910
+       <https://github.com/aboutcode-org/vulnerablecode/pull/1910>`_
+   * - 3
+     - Add Curl Live Importer V2
+     - `aboutcode-org/vulnerablecode#1923
+       <https://github.com/aboutcode-org/vulnerablecode/pull/1923>`_
+   * - 4
+     - Add Elixir Security Live V2 Importer
+     - `aboutcode-org/vulnerablecode#1935
+       <https://github.com/aboutcode-org/vulnerablecode/pull/1935>`_
+   * - 5
+     - Add NPM Live Importer V2
+     - `aboutcode-org/vulnerablecode#1941
+       <https://github.com/aboutcode-org/vulnerablecode/pull/1941>`_
+   * - 6
+     - Add GitHub OSV Live V2 Importer Pipeline
+     - `aboutcode-org/vulnerablecode#1977
+       <https://github.com/aboutcode-org/vulnerablecode/pull/1977>`_
+   * - 7
+     - Add Postgres Live V2 Importer Pipeline
+     - `aboutcode-org/vulnerablecode#1982
+       <https://github.com/aboutcode-org/vulnerablecode/pull/1982>`_
+   * - 8
+     - Add PySec Live V2 Importer Pipeline
+     - `aboutcode-org/vulnerablecode#1983
+       <https://github.com/aboutcode-org/vulnerablecode/pull/1983>`_
+   * - 9
+     - Add Local VulnerableCode Datasource in VulnTotal and allow live evaluation
+     - `aboutcode-org/vulnerablecode#1985
+       <https://github.com/aboutcode-org/vulnerablecode/pull/1985>`_
+   * - 10
+     - Integrate Local VulnerableCode datasource and live evaluation
+     - `aboutcode-org/vulntotal-extension#17
+       <https://github.com/aboutcode-org/vulntotal-extension/pull/17>`_
+
+
+Closing Thoughts
+-------------------
+
+This project was an exciting step forward from my 2024 GSoC work. By moving
+from batch importers to package-first live importers, We enabled a faster,
+more personalized, and more flexible way of building vulnerability databases.
+
+I especially enjoyed designing the **registry + API architecture** and
+integrating Redis queues and workers for asynchronous execution. This improved
+scalability, responsiveness, and fault tolerance, ensuring the API never blocks
+and multiple live evaluations can run in parallel. I also appreciated discussing
+it with mentors and integrating it seamlessly across
+**VulnerableCode, VulnTotal, and the browser extension**.
+
+This work lays the foundation for even richer interactivity
+in the ecosystem and brings vulnerability evaluation closer
+to developers' workflows.
+
+I appreciated the weekly status calls and the feedback I received from my
+mentors and the amazing team. They were really helpful and supportive. -
+`Philippe Ombredanne <https://github.com/pombredanne>`_ - `Ayan Sinha
```

Comment:
If you want a list, you have to start these in new lines, otherwise you can remove the `-` and have the names seperated by commas.